scripts/setup-ci.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+command -v ssh-agent >/dev/null || ( sudo apt-get update -y && sudo apt-get install openssh-client -y )
+eval $(ssh-agent -s)
+
+mkdir -p ~/.ssh
+chmod 700 ~/.ssh
+mv $GITHUB_SSH_KEY ~/.ssh/id_rsa
+chmod 600 ~/.ssh/id_rsa
+ssh-add ~/.ssh/id_rsa
+ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
+
+git config --global user.email colin@konghq.com
```

Comment:
@hutchic the Jenkinsfile diff above refers to a github bot... should be using the bot's info instead of your own?